MetaMorpheus/Test/FdrTest.cs

```diff
@@ -212,7 +218,31 @@ public static void TestComputePEPValue()
                 { Path.GetFileName(maxScorePsm.FullFilePath), 0 }
             };
 
-            var maxPsmData = PEP_Analysis_Cross_Validation.CreateOnePsmDataEntry("standard", fsp, maxScorePsm, fileSpecificRetTimeHI_behavior, fileSpecificRetTemHI_behaviorModifiedPeptides, massError, chargeStateMode, pwsm, notch, !pwsm.Parent.IsDecoy);
+            // Set values within PEP_Analysis through reflection
```

Comment:
Cool use of reflection, what specifically does it help you test here?

Reply:
Previously, we were manually constructing dictionaries and then running individual PSMs through PEP to ensure that the right values came out the other side. Now you have to use reflection to make those dictionaries, as they're no longer public static.